src/logic/quickstart.ts

```diff
@@ -50,62 +59,54 @@ export async function interactiveInstall(
   let envContent = '';
   //Super.json path
   let superPath = await detectSuperJson(process.cwd());
-  if (superPath) {
-    //Overide existing super.json
-    if (
-      !(await confirmPrompt(
-        'Configuration file super.json already exists.\nDo you want to override it?:'
-      ))
-    ) {
-      options?.warnCb?.(`Super.json already exists at path "${superPath}"`);
-
-      return;
-    }
+  if (!superPath) {
+    //Init SF
+    options?.successCb?.('Initializing superface directory');
+    await initSuperface(
+      './',
```

Comment:
maybe use the normalized version? or better yet, save the normalized version to constant and use everywhere?

Reply:
I can't use normalized version everywhere - during process I need to reload super.json (after profile install, add profile defaults). This is because of too high level approach of functions installProfile and installProvider.